src/js/components/Settings/VoterEmailAddressEntry.jsx

```diff
@@ -531,50 +531,50 @@ class VoterEmailAddressEntry extends Component {
             </FirstRowPhoneOrEmail>
             <SecondRowPhoneOrEmail>
               {isPrimaryEmailAddress ? (
-                <div key={`${voterEmailAddressFromList.email_we_vote_id}-internal`}>
+                <SecondRowPhoneOrEmailDiv>
                   <span>
                     Primary
-                    <span style={{ paddingRight: '112px' }}>&nbsp;</span>
                   </span>
-                  <OverlayTrigger
-                    placement="right"
-                    overlay={(
-                      <Tooltip id="tooltip-top">
-                        Cannot remove primary email. Please add another primary email before removing this one.
-                      </Tooltip>
-                    )}
-                  >
-                    <TrashCan>
-                      <span
-                        // className="u-gray-darker"
-                        style={{ color: '#757575' }}
-
-                      >
-                        <Delete />
-                      </span>
-                    </TrashCan>
-                  </OverlayTrigger>
-                </div>
+                  <span>
+                    <OverlayTrigger
+                      placement="right"
+                      overlay={(
+                        <Tooltip id="tooltip-top">
+                          You must add a new primary email before removing this one.
+                        </Tooltip>
+                      )}
+                    >
```

Comment:
Hi @smcvey7 I'm approving this now, but can you please check in an iteration with your next pull request where you continue to use the styled div name for "TrashCan". It helps with code readability.